actionview/lib/action_view/template/renderable.rb

```diff
@@ -14,8 +14,12 @@ def identifier
 
       def render(context, *args)
         @renderable.render_in(context)
-      rescue NoMethodError
-        raise ArgumentError, "'#{@renderable.inspect}' is not a renderable object. It must implement #render_in."
+      rescue NoMethodError => error
+        if error.name == :render_in
```

Comment:
Should we make the condition here `!@renderable.respond_to?(:render_in)`? Otherwise I think it's still _possible_ that this captures a nested error instead of the one we want

Reply:
Good idea!  Updated in 91968e5a185927b8216b82c860a48b6367285456.